pom.xml

```diff
@@ -30,7 +30,7 @@
 		<repository>
 			<id>Sonatype-public</id>
 			<name>SnakeYAML repository</name>
-			<url>http://oss.sonatype.org/content/groups/public/</url>
+			<url>https://oss.sonatype.org/content/groups/public/</url>
```

Comment:
You are right, this can be completely removed. Would you like to add a commit for that? 🙇